### ISSUE TITLE
Inquire about the LOD related matters of IFC

### ISSUE THREAD

**Issue by jingyangking** [ifcquery/ifcplusplus#274]
Hello developer, I would like to inquire if there is a tool that can automatically grade IFC models without the need for format conversion. If no such tools are currently available, can we manually grade the IFC model directly? How to operate it specifically? Thank you.

**Comment by ifcquery** [comment]
Hi,
you mean upgrading an IFC model? For example from 2X3 to 4X3?
That is possible with IFC++, reading a file, and writing to a new file.
But what do you mean with level of detail (LOD)?

**Comment by jingyangking** [comment]
Sorry, I have a problem with the meaning expressed earlier. What I want to say is whether there is a tool that can automatically subtract faces from IFC models to form IFC models with different levels of detail

**Comment by jingyangking** [comment]
Just like how the detail level of the exported IFC model can be set in REVIT. But there are some shortcomings in the surface reduction operation performed by REVIT. Is there a better tool to replace it? thanks

**Comment by jingyangking** [comment]
yes,you are right.How to achieve such an operation? It would be even better if we could customize the surface reduction

**Comment by jingyangking** [comment]
You misunderstood my idea. What I mean is to reduce the overall number of faces of the IFC model, just like the image below, but what I need is to do the overall reduction.
![image](https://github.com/ifcquery/ifcplusplus/assets/112468121/df35d14b-2f56-40f2-9e11-36126615aef4)

**Comment by ifcquery** [comment]
You can model that in IFC with IfcCircle. No need for LOD

**Comment by jingyangking** [comment]
Okay, I will try it out. Thank you for your reply
